fuel-tx/src/lib.rs

```diff
@@ -4,6 +4,7 @@
 // Wrong clippy convention; check
 // https://rust-lang.github.io/api-guidelines/naming.html
 #![allow(clippy::wrong_self_convention)]
+#![deny(clippy::string_slice)]
```

Comment:
should we also add this to the `fuel-types` lib.rs?

Reply:
I'll do that as a follow-up, as I'm alreaady doing parsing changes as a part of #533